cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/AuthenticationSystemSupport.java

```diff
@@ -23,4 +31,53 @@
      * @return the principal election strategy
      */
     PrincipalElectionStrategy getPrincipalElectionStrategy();
+
+    /**
+     * Initiate potential multi-transaction authentication attempt by handling the initial authentication transaction.
+     *
+     * @param credential a credential for the initial authentication transaction.
+     *
+     * @return authentication context builder used to accumulate authentication transactions in this authentication attempt.
+     *
+     * @throws AuthenticationException exception to indicate authentication processing failure.
+     * @since 4.3.0
+     */
+    AuthenticationContextBuilder handleInitialAuthenticationTransaction(Credential... credential) throws AuthenticationException;
```

Comment:
No, you're right, it's really about handling the initial authentication transaction.